test/Microsoft.DotNet.Cli.Utils.Tests/project.json

```diff
@@ -1,8 +1,17 @@
 {
   "version": "1.0.0-*",
-  "compilationOptions": {
+  "buildOptions": {
     "emitEntryPoint": true,
-    "keyFile": "../../tools/Key.snk"
+    "keyFile": "../../tools/Key.snk",
+    "copyToOutput": {
+      "include": [
+        "../../TestAssets/TestProjects/OutputStandardOutputAndError/*",
```

Comment:
I know you didn't design the format - but IMO this points out a design flaw.  How often do you want the "copyToOutput" files copied both on build and on publish?  Most of the time, I would think.  I would also want them in my "pack" as well ...